src/pages/Store/Pickups.vue

```diff
@@ -98,6 +113,55 @@ export default {
       leave: 'pickups/leave',
     }),
   },
+  data: function () {
+    return {
+      routeUrl: 'https://www.openstreetmap.org/directions?engine=graphhopper_bicycle',
+    }
+  },
+  mounted: function () {
+    if (this.store.latitude) {
+      var storeLocation = encodeURI(`${this.store.latitude},${this.store.longitude}`)
+
+      var createOSMUrl = (from) => {
+        if (from) {
+          return `https://www.openstreetmap.org/directions?engine=graphhopper_bicycle&from=${encodeURI(from)}&to=${storeLocation}`
+        }
+        else {
+          return `https://www.openstreetmap.org/directions?engine=graphhopper_bicycle&to=${storeLocation}`
+        }
+      }
+
+      var noGeolocationAvailable = positionError => {
+        if (this.currentUser.address) {
+          this.routeUrl = createOSMUrl(this.currentUser.address)
+        }
+        else {
+          this.routeUrl = createOSMUrl()
+        }
+      }
+
+      var geolocationAvailable = position => {
+        var geolocation = `${position.coords.latitude},${position.coords.longitude}`
+        this.routeUrl = createOSMUrl(geolocation)
+      }
+
+      var detected = browser()
+      if (detected && detected.mobile && detected.os.includes('OS X')) {
+        this.routeUrl = `http://maps.apple.com/?saddr=Current%20Location&daddr=${storeLocation}`
+      }
+      else if (detected && detected.mobile && detected.os.includes('Android')) {
```

Comment:
I don't have access to a real android device so I only tested this code branch on chrome's emulator.